src/components/schema-ui/schema-menu-attribute.js

```diff
@@ -67,7 +67,13 @@ const AttributeFocusToggle = ({path}) => {
 };
 
 const SchemaMenuAttributeValue = ({ name, value, forPath, level }) => {
-  const { type, title, description, properties, ...values } = value;
+  let { type } = value;
+  let { title, description, properties, items, ...values } = value;
+
+  if (type === 'array') {
+    ({ type, properties, ...values } = items);
+    type = `[${type}]`;
```

Comment:
What's the purpose with this? Why destructure type first, then overwrite it as `[object]` for example?

Reply:
If you look at the schema for a field w/ cardinality > 1. The schema object is:

```

Reply:
so the goal is to show the item "type" vs the field type?

Reply:
The goal was to show `[integer]` as the primary type for an integer field with cardinality > 1 instead of:

![Screen Shot 2019-06-25 at 17 20 50](https://user-images.githubusercontent.com/4500265/60140249-516ff800-976e-11e9-9944-7408a66aae9f.png)


But I think I took the wrong tack cause now I'm breaking other things. Let me try something else.